### PR TITLE
Prefab support for bugsnag-plugin-android-ndk

### PR DESCRIPTION
**Pull request by lemnik**
## Goal
Support linking to `bugsnag-plugin-android-ndk` via prefabs.

## Changes

- bugsnag-plugin-android-ndk is now published with a `bugsnag-ndk` prefab module (in addition to the standard `jniLibs`)
- Mazerunner is no longer built with NDK 16.1.4479499 as it fails to generate the `cmake` files for prefabs.

## Testing
Modified the existing Mazerunner fixture to use prefabs instead of manually unpacking the `.so` files

### CODE REVIEW

File: examples/sdk-app-example/app/CMakeLists.txt [hunk 1]
```diff
@@ -11,3 +11,13 @@ set_target_properties(lib_bugsnag PROPERTIES IMPORTED_LOCATION
...
+
+#cmake_minimum_required(VERSION 3.6)
+#
+#find_package(bugsnag-plugin-android-ndk REQUIRED CONFIG)
+#
+#add_library(entrypoint SHARED
+#        src/main/cpp/entrypoint.cpp)
+#
+#target_include_directories(entrypoint PRIVATE ${BUGSNAG_INCLUDE_DIR})
+#target_link_libraries(entrypoint bugsnag-plugin-android-ndk::bugsnag-ndk)
```
Comment (tomlongridge) [inline examples/sdk-app-example/app/CMakeLists.txt:23]: Probably remove?

**Review by tomlongridge — APPROVED** [review]
LGTM - one minor comment (just to show I'm looking!)

**Comment by twometresteve** [comment]
`docs/ENVIRONMENT_SETUP.md` still contains a reference to "r16", I'm guessing that can be removed now?

**Review by twometresteve — APPROVED** [review]
CI aspects LGTM

**Review by SmartbearYing — APPROVED** [review]
LGTM

**Pull request merged**